src/View/View.php

```diff
@@ -497,7 +497,7 @@ public function autoLayout($autoLayout = null)
     /**
      * Get the current view theme.
      *
-     * @return string
+     * @return string|null
```

Comment:
Can the setters then also accept null? E.g. `setTheme($theme)`

Reply:
Yes it can.

Reply:
We should also point that out. One more advantage of separating the concerns in separate methods :)

Reply:
I updated docblock of `setTheme()` too.